tools/ci_build/github/android/build_aar_package.py

```diff
@@ -151,7 +159,7 @@ def parse_args():
     parser.add_argument("--android_ndk_path", type=str, default=os.environ.get("ANDROID_NDK_HOME", ""),
                         help="Path to the Android NDK")
 
-    parser.add_argument('--build_dir', type=pathlib.Path, default=os.path.join(REPO_DIR, 'build_android_aar'),
+    parser.add_argument('--build_dir', type=pathlib.Path, default=os.path.join(REPO_DIR, 'build/android_aar'),
```

Comment:
>type=pathlib.Path [](start = 39, length = 17)

nit: the type pathlib.Path is not consistent with the default value type (str). should we consistently use either Path or str?

Reply:
I think we don't need pathlib here, since most cases this folder does not exist yet, changed to str